server/appservices/subscriptions.go

```diff
@@ -166,7 +166,7 @@ func (a *AppServices) unsubscribe(r *incoming.Request, ownerUserID string, e app
 		return modified, nil
 	}
 
-	return all, utils.ErrNotFound
+	return all, errors.New("You are not subscribed to this notification/s")
```

Comment:
2/5 that we should still `errors.Wrap` `utils.ErrNotFound` so the upper layers can check for that error.

Reply:
mmm, I don't understand what you mean, but I'll change it.

Reply:
@javaguirre see `errors.Cause`